filters/resize.go

```diff
@@ -2,18 +2,20 @@ package filters
 
 import (
 	log "github.com/Sirupsen/logrus"
+	"github.com/zalando-incubator/skrop/parse"
 	"github.com/zalando/skipper/filters"
 	"gopkg.in/h2non/bimg.v1"
-	"github.com/zalando-incubator/skrop/parse"
+	"math"
 )
 
 const (
 	ResizeName = "resize"
 )
 
 type resize struct {
-	width  int
-	height int
+	width      int
+	height     int
+	proportion bool
```

Comment:
I would rename `proportion` to `keepAspectRatio`. It's a common way to refer to this use case, when dealing with images.